### ISSUE TITLE
Auto update interval 

### ISSUE THREAD

**Issue by Judy85** [dauphine-dev/drop-feeds#27]
During several tests with every 15 and every 30 minutes set, Automatic feed updates ran every 3:15-4:40 minutes. I'd like to prevent hammering the feed sites I subscribed to too much.

**Comment by Judy85** [comment]
Additionally, could you please add a possibility to stop the Check feeds when it's running, maybe by clicking the Check feeds icon itself again?

**Comment by dauphine-dev** [comment]
I have created a separate issue https://github.com/dauphine-dev/drop-feeds/issues/30 for 

> Stop the Check feeds when it's running

**Comment by Judy85** [comment]
I tested build 0.7.1299 and the auto update work intermittently. 

Results:
- every 1 minute (prod profile): it ran once after about 40 seconds and then no more;
- every 1 minute (new profile) : after restart of Firefox, it never ran;
- every 5 minutes (new profile): never ran;
- every 5 minutes (prod profile): after restart, ran after 3 minutes and 40 seconds;
- every 15 minutes (prod profile): ran after 4 minutes and 40 seconds already.

I tested both in my prod profile and in a new profile with only drop feeds installed.
